sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -936,6 +941,16 @@ public enum BlobType
         Page = 1,
         Append = 2,
     }
+    public partial class BlobUploadFromUriOptions
+    {
+        public BlobUploadFromUriOptions() { }
+        public Azure.Storage.Blobs.Models.AccessTier? AccessTier { get { throw null; } set { } }
+        public Azure.Storage.Blobs.Models.BlobCopySourceBlobPropertiesOption CopySourceBlobPropertiesOption { get { throw null; } set { } }
```

Comment:
this is boolean on the server side. enum might be overkill.

Reply:
The issue here is that the server side bool is true by default.  This violates One SDK's guidelines.

[Put Blob from URL spec](https://microsoft-my.sharepoint.com/:w:/p/rerdmann/EYisvYQjpmZEp7_kmd4A5ksBaDcX_Oi_q7s7l7Sjkqb9KA?e=gk1F0R&CID=7B4EA60C-B7B5-40E6-88F1-E4E48B031366&wdLOR=cF04C736C-0D95-4669-8761-5139D9BCA270)

> x-ms-copy-source-blob-properties: true/false.  Optional.  Specifies whether the properties of the source blob should be copied to the destination.  This is true by default.  If false, no properties from the source blob are copied, and only those properties provided in the request are applied to the destination blob.

Reply:
@tg-msft should we invert boolean or keep enum ?

Reply:
Could we use `bool? CopyBlobProperties { get; set; }` where the default `null` means do whatever the service wants?  That might also help make it clear customers only need to think about setting it when they're doing a copy vs. uploading a random URI?

Reply:
Sounds good to me. That's how Java equivalent is implemented now.